uiowa/uiowa
# Added additional_person_types split to config_ignore.

Looks like I forgot to add the additional_person_types status to config_ignore. This is now added, and seems to fix the problem with the additional person types getting cleared upon new release.

# How to test

- Pull down CS site locally: blt ds --site=cs.uiowa.edu
- Enable the additional_person_types feature split under [https://cs.local.drupal.uiowa.edu/admin/config/development/configuration/config-split](https://cs.local.drupal.uiowa.edu/admin/config/development/configuration/config-split).
- Run: drush @cs.local cim
- Ensure that people show up properly under the People > Adjunct Faculty and People > Postdoctoral Researchers pages.
- Run: drush @cs.local cst
- Should get a "notice: No differences between DB and sync directory." in command line.
